src/form-builder-admin-card-link.component.tsx

```diff
@@ -2,16 +2,15 @@ import React from "react";
 import { useTranslation } from "react-i18next";
 import { Layer, ClickableTile } from "@carbon/react";
 import { ArrowRight } from "@carbon/react/icons";
+import { navigate } from "@openmrs/esm-framework";
 
 const FormBuilderCardLink: React.FC = () => {
   const { t } = useTranslation();
   const header = t("manageForms", "Manage Forms");
   return (
     <Layer>
       <ClickableTile
-        href={`${window.spaBase}/form-builder`}
-        target="_blank"
-        rel="noopener noreferrer"
+        onClick={() => navigate({ to: `\${openmrsSpaBase}/form-builder` })}
```

Comment:
@donaldkibet won’t this open the form builder in the same tab as the app? If yes then that is not the desired behaviour because the reason we moved them to the admin is because we want to open them in a new tab.

Reply:
That's fine. In that event i will close this PR.